qiskit_aqua/utils/subsystem.py

```diff
@@ -22,23 +22,39 @@
 from qiskit.tools.qi.qi import partial_trace
 
 
-def get_subsystem_statevector(statevector, trace_systems):
+# def get_subsystem_statevector(statevector, trace_systems):
```

Comment:
I would delete this function as does not work in the general case

Reply:
done!